src/bootstrap/dist.rs

```diff
@@ -456,7 +456,7 @@ impl Step for DebuggerScripts {
     type Output = ();
 
     fn should_run(run: ShouldRun<'_>) -> ShouldRun<'_> {
-        run.path("src/lldb_batchmode.py")
+        run.path("src/etc/lldb_batchmode.py")
```

Comment:
I think we should probably drop support for this being invoked from the CLI. It probably should in general be moved into the sequence of copies we always execute when building a sysroot (Step for Assemble), rather than being ad-hoc managed on demand here. That'll ensure we always have them in every sysroot. We'll probably still need kind of similar logic to the below to make sure they end up in dist tarballs, but I think this approach sounds cleaner to me overall.

(This seems like work for a separate PR, though; if you can file an issue, that'd be great).

Reply:
Makes sense :) I opened https://github.com/rust-lang/rust/issues/95992.